core/src/main/scala/cats/std/tuple.scala

```diff
@@ -0,0 +1,15 @@
+package cats
+package std
+
+trait TupleInstances extends Tuple2Instances
```

Comment:
I think this will cause merge conflicts with #867, but I suppose we can just cross that bridge when we get to it.